postgres/codenamemap.yaml

```diff
@@ -10,25 +11,26 @@
 
 {% macro debian_codename(name, version, codename=none) %}
 
-  {# use upstream version if configured #}
   {% if repo.use_upstream_repo == true %}
     {% set version = repo.version %}
     {% set fromrepo = repo.fromrepo|default(name ~ '-pgdg', true) %}
   {% else %}
     {% set fromrepo = name %}
   {% endif %}
+  {% set conf_dir = '/etc/postgresql/' + version|string + '/main' %}
+  {% set data_dir = '/var/lib/postgresql/' + version|string + '/main' %}
 
 {{ codename|default(name, true) }}:
   # PostgreSQL packages are mostly downloaded from `main` repo component
+  conf_dir: {{ conf_dir }}
+  data_dir: {{ data_dir }}
```

Comment:
Since these values become global for all Debian derivatives, we should move them to the OS family map.

Reply:
The `version` calculated by oscodename is passed to `fedora_codename` macro and used  ..
```
  {% set conf_dir = '/etc/postgresql/' + version|string + '/main' %}
  {% set data_dir = '/var/lib/postgresql/' + version|string + '/main' %}
```
If version calculated by OS map is different to version calculated here, they maybe problems?  What do you think?

Reply:
Oh, you're right, absolutely. I've completely forgot that the version for distro-provided packages is handled here as well. Thanks, keep insisting on right things! :+1: